x-pack/plugins/canvas/server/usage/custom_element_collector.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { get } from 'lodash';
+import { fromExpression } from '@kbn/interpreter/common';
+import { CUSTOM_ELEMENT_TYPE } from '../../common/lib/constants';
+import { collectFns } from './collector_helpers';
+
+function parseJsonOrNull(str) {
+  try {
+    return JSON.parse(str);
+  } catch (e) {
+    return null;
+  }
+}
+
+export function handleResponse({ hits }) {
+  const customElements = get(hits, 'hits', []).map(hit => hit._source[CUSTOM_ELEMENT_TYPE].content);
+  const functionSet = new Set();
+
+  const parsedContents = customElements.map(parseJsonOrNull).filter(contents => contents !== null);
+
+  if (parsedContents.length === 0) {
+    return {};
+  }
+
+  const elements = {
+    min: Infinity,
+    max: -Infinity,
```

Comment:
Interesting... shouldn't this be `min: 0`, `max: Infinity`?

Reply:
This method is trying to find the min and max number elements used in a CustomElement  

We initialize the min Infinity so that the first custom element will have a number of elements fewer than Infinity and becomes the new baseline for minimum.  Vice-versa for max (though I guess it could just be 0 instead of -Infinity).  Really I just borrowed this logic from what was already being done in the current telemetry collection, which now lives in the workpad_collector file.